src/recon_buildblock/PLSPrior.cxx

```diff
@@ -105,17 +112,10 @@ PLSPrior<elemT>::post_processing()
 {
   if (base_type::post_processing()==true)
     return true;
-  if (kappa_filename.size() != 0)
-    this->kappa_ptr = read_from_file<DiscretisedDensity<3,elemT> >(kappa_filename);
```

Comment:
so, keep the if, and replace the next line with `set_kappa_filename` (which will then read)